project-eden/eden-web
# Server/initial setup

To test:

1. On `/fakeServer` run `python fakeApi.py`, wait for the log `Listening on http://localhost:8080/`. The server is ready.
1. On `/eden-web` run `npm start`.
2. Click on a spot on the map.
3. A label showing the top label at that grid will appear. At this point those labels can be numbers since the function is interpreting climatic variables as crops. Our actual table will not contain climatic variables.
